specification/record-locator/consumer.yaml

```diff
@@ -302,7 +302,7 @@ paths:
                       date: "2022-12-20T09:45:41+11:00"
                       author:
                         - identifier:
-                            value: Practitioner/A985657ZA
+                            value: Organization/Y05868
```

Comment:
Given the new validation requires the ods-code system/value in `author.identifier` and there can be only one, should the `author` examples follow the same format as custodian?

```
author:
  identifier:
    system: https://fhir.nhs.uk/Id/ods-organization-code
    value: Y05868
```